lib/hatchet/app.rb

```diff
@@ -495,6 +495,9 @@ def create_source
 
     def delete_pipeline(pipeline_id)
       api_rate_limit.call.pipeline.delete(pipeline_id)
+    rescue Excon::Error::Forbidden
+      warn "Error deleting pipeline id: #{pipeline_id.inspect}, status: 403"
+      # Means the pipeline likely doesn't exist, not sure why though
```

Comment:
@schneems There is a pipeline cleaner that removes any pipelines that contain zero apps, shortly after an app in a pipeline is deleted. I would imagine the pipeline delete races with that. In these cases the pipeline delete could just be skipped entirely, since the pipeline cleaner will take care of it.

Reply:
Good call. Makes sense.